lib/client.js

```diff
@@ -311,7 +347,48 @@ Client.prototype = {
     }.bind(this));
   },
 
+  /// #### client.instance(instanceGuid)
+  ///
+  /// Create an Client wrapper for another app instance.
+  ///
+  /// ##### Arguments
+  ///
+  ///   * `instanceGuid` the GUID of the other instance of your app
+  ///
+  /// ##### Returns
+  ///
+  /// A [Promises/A+](https://promisesaplus.com) conformant `promise` object.
+  ///
+  /// ```javascript
+  /// var client = ZAFClient.init();
+  ///
+  /// client.get('instances').then(function(data) {
+  ///   var guid = Object.keys(data.instances).filter(function(guid) {
+  ///     return data.instances[guid].location == 'top_bar';
+  ///   })[0];
+  ///   return guid;
+  /// }).then(function(guid) {
+  ///   return client.instance(guid);
```

Comment:
Would be nice if this returned a client rather than a promise, there's a fair amount of boilerplate already.